mk/dev.mk

```
@@ -12,6 +12,10 @@ KUMA_CHARTS_URL ?= https://kumahq.github.io/charts
 CHART_REPO_NAME ?= kuma
 PROJECT_NAME ?= kuma
 
+ifdef CI
+	CI_TOOLS_VERSION=master
```

Just to avoid any future confusion/mistake can we:
```suggestion
        # tools version is irrelevant on CI as we checkout only 1 branch at a time.
	CI_TOOLS_VERSION=ci
```